frontend/resume/src/styles/styles.css

```
@@ -234,4 +234,142 @@ footer {
+    justify-content: space-between;
+    align-items: center;
+    margin-top: 10px;
+}
+
+.char-counter {
+    font-size: 8pt;
+    color: #999;
+}
+
+#summarize-btn {
+    font-family: "Raleway", sans-serif;
```

Stylelint (17.14.0)

[error] Expected no quotes around "Raleway" (font-family-name-quotes)

(font-family-name-quotes)